macros/ToolsSidebar.ejs

```diff
@@ -13,13 +13,13 @@ const text = mdn.localStringMap({
       "Network_Monitor": "Network Monitor",
       "Performance": "Performance",
       "Responsive_Design_Mode": "Responsive Design Mode",
+      "Accessibility_inspector": "Accessibility inspector",
```

Comment:
In the other entries we capitalize "inspector".

Reply:
Good point, thanks @wbamberg. I've fixed this in all cases except for in the actual page slug, as I can't fix that now.